holopy/core/io/io.py

```diff
@@ -380,6 +380,96 @@ def save_image(filename, im, scaling='auto', depth=8):
         pilimage.fromarray(im).save(filename)
 
 
+def save_images(filenames, ims, scaling='auto', depth=8):
+    """
+    Saves a volume as separate images (think of reconstruction volumes).
+
+    Parameters
+    ----------
+    ims : ndarray or :class:`holopy.image.Image`
+        Images to save, with separate z-coordinates from which they each will
+        be selected.
+    filename : str
```

Comment:
In my personal opinion, I think a better practice would be to pass in an iterable of filenames, rather than a format string. I think the format string is a little less intuitive and creates more potential for problems.

Reply:
Done!